spec/features/admin/index_spec.rb

```diff
@@ -0,0 +1,72 @@
+require "rails_helper"
+
+RSpec.describe "Admin Shelter Index", type: :feature do
+  before(:each) do
+    @shelter_1 = Shelter.create!(
+      foster_program: true,
+      name: "The Shelter",
+      city: "Happy City",
+      rank: 1
+    )
+    @shelter_2 = Shelter.create!(
+      foster_program: true,
+      name: "City Shelter",
+      city: "Sad City",
+      rank: 1
+    )
+    @shelter_2 = Shelter.create!(
+      foster_program: true,
+      name: "Zoo Shelter",
+      city: "Zoo City",
+      rank: 1
+    )
+    @pet_1 = @shelter_1.pets.create!(
+      adoptable: true,
+      age: 3,
+      breed: "orange cat",
+      name: "Cheesecake"
+    )
+    @application = Application.create!(
+      name: "John Smith",
+      street_address: "1234 Lane Street",
+      city: "Happy City",
+      state: "CO",
+      zip_code: "80111",
+      owner_description: "I want an animal",
+      status: "Pending"
+    )
+  end
+  describe "as a visitor" do
+    describe "when I visit the admin shelter index ('/admin/shelters/)" do
+      it "I see all shelters in the system listed in reverse alphabetical order by name" do
+        visit "/admin/shelters"
+
+        expect(page).to have_content("The Shelter")
+        expect(page).to have_content("City Shelter")
+        expect(page).to have_content("Zoo Shelter")
+
+        expect("Zoo Shelter").to appear_before("The Shelter")
+        expect("The Shelter").to appear_before("City Shelter")
+      end
+    end
+  end
+
+  describe "as a visitor" do
+    describe "when I visit the admin shelter index ('/admin/shelters')" do
+      describe "then I see a section for 'Shelters with Pending Applications'" do
+        it "and In this section I see the name of every shelter that has a pending application" do
+          @application.pets << @pet_1
+
+          visit "/admin/shelters"
+
+          expect(page).to have_content("Shelters with Pending Applications")
+
+          within("#pending-applications") do
+            expect(page).to have_content("The Shelter")
+            expect(page).not_to have_content("A Shelter")
```

Comment:
What is "A Shelter"? Did you mean "City Shelter" or "Zoo Shelter"? Testing for those would make this test more robust because right now I don't have confirmation that the other created shelters don't show up